tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -73,7 +73,7 @@ Amount_Remittance_OutwardRemittance,Amount of Outward Remittance
 Amount_Stock,Stock market capitalization
 Amount_Stock_AsFractionOf_Amount_EconomicActivity_GrossDomesticProduction_Nominal,Stock market capitalization as a Proportion of Nominal GDP
 AmountFarmInventory_WinterWheatForGrain,Amount of Winter Wheat grown for Grain production
-Amout_FarmInventory_CornForGrain,Amount of Corn grown for grain production
+Amount_FarmInventory_CornForGrain,Amount of Corn grown for grain production

```

Comment:
When you make this change, you also need to:
- re-build embeddings: https://github.com/datacommonsorg/website/blob/master/tools/nl/embeddings/README.md#build-embeddings-index
- follow these steps to validate the embeddings & include the right files in this PR: https://github.com/datacommonsorg/website/blob/master/tools/nl/embeddings/README.md#build-embeddings-index